src/session/message_loop.js

```diff
@@ -45,7 +45,18 @@ class MessageLoop {
             clearImmediate: (t) => { if (t instanceof SessionClearableTimer) t._clear(); else clearImmediate(t) }, 
             clearInterval: (t) => { if (t instanceof SessionClearableTimer) t._clear(); else clearInterval(t) }, 
             clearTimeout: (t) => { if (t instanceof SessionClearableTimer) t._clear(); else clearTimeout(t) },
-            exports, module, require,
+            exports, module, 
+            require: (id) => {
```

Comment:
Node [require](https://nodejs.org/api/modules.html#modules_require_id) docs (for reference)